src/utils/filter.ts

```diff
@@ -0,0 +1,13 @@
+import { Todo } from '../types/Todo';
+import { Filter } from '../types/Filter';
+
+export const filter = (todos: Todo[], type: Filter) => {
```

Comment:
Consider using a more descriptive name for the function. `filter` is too generic and may cause naming conflicts. 🤔